tests/unit/dataactvalidator/test_b11_object_class_program_activity_3.py

```diff
@@ -24,6 +24,12 @@ def test_success(database):
 def test_failure(database):
     """ Test invalid object class code (3 digits) """
 
+    # This should return because if it's '000', '00', '0' a warning should be returned
     op = ObjectClassProgramActivityFactory(object_class='000')
+    assert number_of_errors(_FILE, database, models=[op]) == 1
+
+    op = ObjectClassProgramActivityFactory(object_class='00')
+    assert number_of_errors(_FILE, database, models=[op]) == 1
 
+    op = ObjectClassProgramActivityFactory(object_class='0')
```

Comment:
We don't need to separate these into different asserts. You can just set each of them to op_1, op_2, and op_3 and then do this:
`assert number_of_errors(_FILE, database, models=[op_1, op_2, op_3]) == 3`
Same for the other test file